lib/pf/node.pm

```diff
@@ -296,7 +296,9 @@ sub node_db_prepare {
     ];
 
     $node_statements->{'node_expire_unreg_field_sql'} = get_db_handle()->prepare(
-        qq [ select mac from node where status="reg" and unregdate != 0 and unregdate < now() ]);
+        qq [ select mac from node where ( 
+                status="reg" and unregdate != 0 and unregdate < now() ) or (
```

Comment:
i feel very picky but can you put the opening parenthesis on the line below and same for the line above. That would make it clearer to read IMO